src/connectors/WalletConnectV2Connector.ts

```diff
@@ -0,0 +1,122 @@
+import { ConnectorUpdate } from '@web3-react/types'
+import EthereumProvider from '@walletconnect/ethereum-provider'
+import { ChainId } from '@dcl/schemas'
+import { AbstractConnector } from './AbstractConnector'
+
+export type WalletConnectV2ConnectorConfig = Parameters<
+  typeof EthereumProvider.init
+>[0]
+
+export class WalletConnectV2Connector extends AbstractConnector {
```

Comment:
The other connectors [seem to be tested](https://github.com/decentraland/decentraland-connect/blob/master/test/connectors.spec.ts) would you mind adding tests for this connector as well?